.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+      IS_CICD: true
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8.6.12
+      - name: Setup nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: "pnpm"
+
+      - name: Install dependencies
+        run: pnpm install
+
+      - name: Configure Git
+        run:
+          git config --global user.email "pcc@pantheon.io" &&  git config
+          --global user.name "PCC"
+
+      - name: Test
+        run: CI=false && pnpm --filter '*cli*' test
```

Comment:
We can remove `CI=false` once we remove `pnpm-lock.yaml` file from Gatsby starter kit. 
We should not have lock files inside starter kits, because we want `install` command to pull the latest dependencies as per `package.json`.

Reply:
It looks like you've already removed it?

Reply:
Yeah, but it's not pushed to GitHub repo from which the `pcc init` command pulls the starter kit.